mergify_engine/tests/conftest.py

```diff
@@ -71,6 +71,19 @@ async def redis_stream() -> typing.AsyncGenerator[utils.RedisStream, None]:
             await utils.stop_pending_yaaredis_tasks()
 
 
+@pytest.fixture()
+async def redis_train() -> typing.AsyncGenerator[utils.RedisTrain, None]:
+    with utils.yaaredis_for_train() as client:
+        await client.flushdb()
+        await redis_utils.load_scripts(client)
```

Comment:
Do we need to load all scripts?

Reply:
mypy should have complain about this one.

Reply:
Without it, I counldn't generate the fake_repository fixture which made some tests run with errors

Reply:
This hides another issue I think.

Also, Mypy is sick here, `load_scripts` takes a `RedisCache` while we pass a  `RedisTrain`. This should have raised an error. I wonder why it didn't catch it.